### PR TITLE
add 不背单词 loader

### PR DESCRIPTION
**Pull request by Zxilly**
https://github.com/yihong0618/GitHubPoster/issues/44 的一部分

### CODE REVIEW

**Comment by yihong0618** [comment]
我周一看一下(^-^)

**Review by umm233 — COMMENTED** [review]
LGTM maybe some change?

File: github_poster/cli.py [hunk 1]
```diff
@@ -52,7 +52,6 @@ def main():
     # set animate
     p.set_with_animation(args.with_animation)
     p.set_animation_time(args.animation_time)
-    p.units = args.loader.unit
```
[the remark this answers is not in the document]
Comment (Zxilly) [reply github_poster/cli.py:55]: for https://github.com/yihong0618/GitHubPoster/pull/45/files#diff-d5914ffbda27b9712fc775e9d19c7f7a4d14f3db519010212b993d4d26d0bdfcR117 
dynamic change the unit of loader. So I lazily load unit from loader to implement this.

File: github_poster/loader/bbdc_loader.py [hunk 2]
```diff
@@ -37,48 +51,53 @@ def update_cache(self):
 
         data_path = os.path.join(os.getcwd(), "data")
         cache_path = os.path.join(os.getcwd(), "data", "bbdc.json")
```
Comment (yihong0618) [inline github_poster/loader/bbdc_loader.py:50]: can we also strore it in `IN_FOLDER`?

**Review by yihong0618 — COMMENTED** [review]
LGTM

**Comment by yihong0618** [comment]
@Zxilly 谢谢

先合并了，我感觉额外的文件都放在 `IN_FOLDER` 里好一些，你觉得呢？

**Pull request merged**